src/Microsoft.ML.AutoML/AutoMLExperiment/AutoMLExperiment.cs

```diff
@@ -296,22 +296,8 @@ void handler(object o, EventArgs e)
                         tuner.Update(trialResult);
                         trialResultManager?.AddOrUpdateTrialResult(trialResult);
                         aggregateTrainingStopManager.Update(trialResult);
```

Comment:
We need to update tuner/trialresultmanager/trainningStopManager even when a trial fails for any reasons as long as the experiment still continues